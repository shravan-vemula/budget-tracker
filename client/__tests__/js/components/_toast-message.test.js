import React from 'react';
import {cleanup} from '@testing-library/react';
import ToastMessage from 'js/_components/atoms/_toast-message';

afterEach(cleanup)


describe("BTCheckbox Component",()=>{
 
    it("renders correctly", () => {
        const wrapper = shallow(<ToastMessage />);
        expect(wrapper).toMatchSnapshot();
      });


});