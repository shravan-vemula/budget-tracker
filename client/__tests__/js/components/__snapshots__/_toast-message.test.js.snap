// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BTCheckbox Component renders correctly 1`] = `ShallowWrapper {}`;
